CHANGELOG.md

```diff
@@ -5,7 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
-<!-- ## Unreleased -->
+## Unreleased
+- When injecting html imports into documents for inlining, bundler now tries
```

Comment:
"When inlining HTML imports, preserve the ordering of imports between inlined and referenced documents"?